.github/workflows/deploy_stage.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        relayer_id: [0, 1, 2, 3, 4]
```

Comment:
I do not like that we are basically having copy of this file just to deploy to another region. Might we use if statements with matrix values? like if matrix.relayer_id.3 deploy to this region, etc. 
Or for example we can use `include` statement of GA that allwos to add additional arguments to particular matrix elements. And we can use it to specify region per relayer_id